__tests__/View-tests/home/homeScreen.test.tsx

```diff
@@ -161,4 +169,21 @@ describe("HomeScreen UI Tests", () => {
     // Vérifie que les défis et groupes ne sont pas affichés
     expect(getByText("No challenges to display")).toBeTruthy();
   });
+
+  it("likes a post on double-tap in HomeScreen", () => {
```

Comment:
I don't think this test does what the description says it does. Are you sure there wasn't a mismatch somewhere here ?

Reply:
Oh right thank you ! Took a break in the middle of coding and forgot the end of the test.